notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py

```diff
@@ -107,27 +108,9 @@
 low_passed = glp.run(deconvolved, in_place=False, verbose=True)
 # EPY: END code
 
-# EPY: START markdown
-#Use MERFISH-calculated size factors to scale the channels across the imaging rounds and visualize the resulting filtered and scaled images. Right now we have to extract this information from the metadata and apply this transformation manually.
-# EPY: END markdown
-
-# EPY: START code
-scale_factors = {
-    (t[Indices.ROUND], t[Indices.CH]): t['scale_factor']
-    for index, t in primary_image.tile_metadata.iterrows()
-}
-# EPY: END code
-
 # EPY: START code
-# this is a scaling method. It would be great to use image.apply here. It's possible, but we need to expose H & C to
-# at least we can do it with get_slice and set_slice right now.
-from copy import deepcopy
-scaled_image = deepcopy(low_passed)
-
-for indices in primary_image._iter_indices():
-    data = scaled_image.get_slice(indices)[0]
-    scaled = data / scale_factors[indices[Indices.ROUND.value], indices[Indices.CH.value]]
-    scaled_image.set_slice(indices, scaled)
+sc_filt = ScaleByPercentile(p=90)
+scaled_image = sc_filt.run(low_passed, in_place=False)
```

Comment:
I thought our default is supposed to be in_place=False now, but it looks like this is not the case for ScaleByPercentile...?

Reply:
Yea scale by percentile still has in_place=True default, but I could change it?

Reply:
oh hm, it's still True everywhere.  

@ambrosejcarr : what happened re https://github.com/spacetx/starfish/issues/384?  It was closed but why?

Reply:
I misinterpreted the ticket -- I closed it because out-of-place is now _possible_ everywhere. Thanks for finding this, will reopen and remember to leave more specific messages.